### ISSUE TITLE
Feed back into glottolog

### ISSUE THREAD

**Issue by SimonGreenhill** [lexibank/lexibank#193]
We should push lexibank information back into glottolog e.g.

- contents of languages.csv as `Alternative Names`
- source information
- A lexibank dataset could go into a new glottolog resource ('dataset'?) and be added as a reference/source for each language in the dataset?

**Comment by xrotwang** [comment]
Yes, we should do this. The process must be automated, though. Otherwise we get into sync-issues. So `languages.csv` (or the source data) is the authoritative source of the Glottolog mapping, and Glottolog then regularly harvests this info. We already have tools in place to update a languoids'
- alternative names (here then for a new section `lexibank`),
- links (here for the domain `lexibank.clld.org`)
- references (here an automatically update file `lexibank.bib`)
